doc/nrf/ug_nrf5340.rst

```diff
@@ -335,6 +335,10 @@ To program a HEX file after building it with |SES|, open a command prompt in the
 
     west flash
 
+If you'd like to erase the whole flash prior to programming, you can use::
+
+    west flash --erase
+
 If you prefer to use nrfjprog (which is part of the `nRF Command Line Tools`_) instead, open a command prompt in the build folder of the network sample and enter the following commands to program the network sample::
 
     nrfjprog -f NRF53 --coprocessor CP_NETWORK --eraseall
```

Comment:
I think we should no longer suggest doing the erase operation separately. I for some reason the pin reset or power cycle occurs before the core is programmed, it will get locked.
Perhaps the programming command should be instead changed to:
```
nrfjprog -f NRF53 --coprocessor CP_NETWORK --program zephyr/zephyr.hex --chiperase
```
Or maybe `--sectorerase` should be added? Why is the full erase actually performed here?

Reply:
Agreed. I will add both options (sector/all)